src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SideMenuView/SideMenuView.shared.cs

```diff
@@ -87,6 +87,17 @@ public class SideMenuView : BaseTemplatedView<AbsoluteLayout>
 		public static readonly BindableProperty MenuGestureEnabledProperty
 			= BindableProperty.CreateAttached(nameof(GetMenuGestureEnabled), typeof(bool), typeof(SideMenuView), true);
 
+		public SideMenuView()
+		{
+#if __ANDROID__
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new Xamarin.CommunityToolkit.Android.UI.Views.SideMenuViewRenderer(null);
+#elif __IOS__
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new Xamarin.CommunityToolkit.iOS.UI.Views.SideMenuViewRenderer(null);
+#endif
```

Comment:
I don't think that we need that here since we are using the Preserve Attribute

Reply:
What does it means negative Ticks here ?

Reply:
Hm, are you 100 percent sure?
I am not an expert in setting the linker :)

That's what I found in docs

> Sometimes you want to preserve certain members, but only if the containing type was preserved. In those cases, use [Preserve (Conditional=true)]

I read it as "linker won't remove filed/method/property IF the entire class is not removed as well"

Reply:
> What does it means negative Ticks here ?

That's a trick) this condition is always "false", but we cannot put just ```if (false)``` because the linker/compiler will remove the body of the statement.

Reply:
@AndreiMisiukevich, I sent a message on discord, but just to document here. Since you're calling the Renderer ctor in code, in a static way the linker will not remove the Renderer.